examples/step-51/doc/intro.dox

```diff
@@ -29,16 +29,9 @@ for systems of equations in 2 or 3 dimensions.
 <h4> Reducing the size of the linear system </h4>
 To alleviate the computational cost of solving such large linear systems,
 the hybridizable discontinuous Galerkin (HDG) methodology was introduced
-by Cockburn and co-workers
-(see the references in the recent HDG overview article by
- N.C. Nguyen and J. Peraire:
-   <i>Hybridizable discontinuous Galerkin methods for partial differential
-   equations in continuum mechanics</i>, Journal of Computational Physics,
-   2012, 231:18, 5955-5988.
-   <a href="http://dx.doi.org/10.1016/j.jcp.2012.02.033">[DOI]</a>).
-The HDG method achieves
-this goal by formulating the mathematical problem using Dirichlet-to-Neumann
-mappings.  The partial differential equations are first written as a first
+by Cockburn and co-workers (see the references in the recent HDG overview article by Nguyen and Peraire @cite NP12).
```

Comment:
We might get merge conflicts -- I think someone already moved this reference to the .bib file.  But go ahead and we'll deal with it as it happens.

Reply:
The change that also addresses this place is #8476.